src/new/connection.js

```diff
@@ -97,6 +98,18 @@ class Connection extends EventEmitter {
     this.websocket.on('is_online', this.onConnectionChange);
     this.websocket.on('wallet', this.handleWalletMessage);
 
+    this.websocket.on('reload_data', () => {
+      this.emit('reload-data');
+    });
+
+    this.websocket.on('height_updated', (height) => {
+      this.emit('height-updated', height);
+    });
+
+    this.websocket.on('addresses_loaded', (data) => {
+      this.emit('addresses-loaded', data);
```

Comment:
Why do we have to propagate this event? What does it mean for the app?

Reply:
I got it. Shouldn't we rename it to something like `wallet-load-partial-update`?

Reply:
Done